etcdserver/security/security.go

```diff
@@ -116,7 +116,6 @@ func NewStore(server doer, timeout time.Duration) *Store {
 		server:  server,
 		timeout: timeout,
 	}
-	s.ensureSecurityDirectories()
 	s.enabled = s.detectSecurity()
```

Comment:
hmm... this is not super reliable today. etcdserver applies all the previous commands after it starts. And the security new happens concurrently with the starting of etcdserver as far as i can see.